eidas_node/connector/static/eidas_node/connector/formautosubmit.js

```diff
@@ -1,6 +1,6 @@
 function formAutoSubmit () {
     var form = document.querySelector('form.auto-submit')
-    if (!form) {
+    if (!form || window.localStorage && window.localStorage.getItem('disable-form-auto-submit')) {
```

Comment:
This was helpful during testing. I think it's ok to keep it for @jtalir to see the branding.

Reply:
If you disable JS, you can achieve the same thing...

Reply:
But NIA didn't work well without JavaScript.

Reply:
Then you can turn it back on after you see this screen :)

Reply:
Ok, removed :-)